### PR TITLE
fixup travis-ci

### PR DESCRIPTION
**Pull request by mayli**
now travis-ci will run `make docker` for sanity check

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,34 +3,26 @@ services:
...
-      - parted
-      - debootstrap
-      - python-termcolor
-      - python-jsonschema
-      - python-docopt
-      - python-yaml
+      - wget
+      - unzip
       - python-pip
+      - qemu-system-x86
```
Comment (RyanSquared) [inline .travis.yml]: if we're using docker this shouldn't be necessary

if it is necessary, find out why, and remove it

File: .travis.yml [hunk 2]
```diff
@@ -3,34 +3,26 @@ services:
...
-  - "[ -d ~/bootstrap-vz/.git ] || git clone https://github.com/hashbang/bootstrap-vz.git ~/bootstrap-vz"
-  - git -C ~/bootstrap-vz pull --ff-only
-  - sudo -H pip install fysom pyrfc3339
+  - test -f /usr/local/bin/packer || (wget -O packer.zip https://releases.hashicorp.com/packer/1.2.4/packer_1.2.4_linux_amd64.zip && unzip -o packer.zip && sudo mv packer /usr/local/bin/)
+  - sudo -H pip install 'ansible>=2.5'
   
 script:
-  - "[ -d ./etc/.git ] || git clone https://github.com/hashbang/shell-etc.git ./etc"
-  - sudo $HOME/bootstrap-vz/bootstrap-vz manifest.yml
+  - sudo make docker
```
Comment (RyanSquared) [inline .travis.yml]: sudo should not be required if it can be avoided, is there a reason why it is required?

File: .travis.yml (hunk 1 above)
Comment (mayli) [reply .travis.yml]: the Makefile looks for qemu-system-x86 on all targets

File: .travis.yml (hunk 2 above)
Comment (mayli) [reply .travis.yml]: well, I am not sure if the user travis is in docker group or not, I can try removing it.

**Pull request merged**